replica/replica_test.go

```diff
@@ -1004,3 +1007,84 @@ func (s *TestSuite) TestPartialReadZeroEndOffset(c *C) {
 
 	byteEquals(c, expected, buf)
 }
+
```

Comment:
some comments to understand what testcase it is about, even before getting into code to get context

Reply:
added comment